plastron/http.py

```diff
@@ -2,11 +2,9 @@
 import os
 import requests
 import threading
-import time
 from base64 import urlsafe_b64encode
 from collections import namedtuple
-from jwcrypto.jwk import JWK
-from jwcrypto.jwt import JWT
+from plastron.auth.auth import AuthFactory
```

Comment:
The `auth.py` file can be moved from `plastron/auth/auth.py` to `plastron/auth.py` (and the `auth/__init__.py` can be deleted). This will change this import to be the shorter `from plastron.auth import AuthFactory`.